krux_boto/util.py

```diff
@@ -25,6 +25,8 @@
 
 from krux.logging import get_logger
 
+ACCEPTED_DOMAINS = ['krxd.net']
+DEFAULT_DOMAIN = 'krxd.net'
```

Comment:
You're repeating a string still - there's an easy way to avoid that by doing:

```
DEFAULT_DOMAIN = 'krxd.net'
ACCEPTED_DOMAINS = [ DEFAULT_DOMAIN ]
```

Question for @hanpeter - are we doing only a single 'accepted domain on purpose? Remember, this is an open source library...